lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.Model.rb

```diff
@@ -2182,6 +2272,7 @@ def model_get_bat_wwr_target(bat, wwr_list)
   # @param total_fene_m2 [Float] total fenestration area
   # @return [Float] reduction factor
   def model_get_wwr_reduction_ratio(multiplier,
```

Comment:
method should be surface_get_wwr_reduction_ratio.
And then just pass in the surface. You pass in name, and results from methods with surface as an argument.

Reply:
Do you suggest to just pass in a surface and then get `surface_name`, `surface_wwr` and `surface_dr` shall be a local variable inside the `model_get_wwr_reduction_ratio`?

Reply:
This function has updated to `surface_get_wwr_reduction_ratio` with `surface` as a new argument to replace `surface_name`, `surface_wwr` and `surface_dr`